peachjam/static/stylesheets/_global.scss

```diff
@@ -91,7 +91,7 @@ footer {
 }
 
 .breadcrumb {
-  margin-top: map-get($spacers, 1);
+  margin: 0;
```

Comment:
LOL, maybe just set the spacing here and drop `mt-3` on each `.breadcrumb`?

Reply:
done. Used padding-top instead of margin-top because margin collapsing issue